delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/http/HttpClient.scala

```diff
@@ -76,7 +76,7 @@ object HttpClient {
 
   private[http] object HttpSingleRequest {
     def default(implicit as: ActorSystem): HttpSingleRequest =
-      (request: HttpRequest) => IO.fromFuture(IO.delay(Http().singleRequest(request)))
+      (request: HttpRequest) => IO.fromFuture(IO.blocking(Http().singleRequest(request)))
```

Comment:
It feels like this might not be blocking? If you take out the future, which should be dealt with by `fromFuture`?

Reply:
Think you're right, you'd hope `fromFuture` deals with the fact that this may be run on an external threadpool like Akka's one

Reply:
I wasn't worried about the future itself but things that could surround it.
But it is true that Akka in particular should handle this correctly and in the implementation, it just send a message mode so it should be fine